tests/hook_main.py

```diff
@@ -734,8 +734,8 @@ def feedback_post(transaction):
     :return:
     """
     with stash['app'].app_context():
-        event = EventFactoryBasic()
-        db.session.add(event)
+        feedback = FeedbackFactory()
+        db.session.add(feedback)
```

Comment:
You should use SessionFactory

Reply:
Using SessionFactory is failing tests. Moreover all hooks having creation object e.g. user, event, role etc. are followed similarly. I think it'll be okay to use this only @iamareebjamal 
Your views?

Reply:
You changed event feedback to session feedback. Instead, we are supporting both if I'm not wrong

Reply:
@iamareebjamal yes we are supporting both but currently only session feedback is being used. Moreover the error raised was related to user, not event.` User.id = 2 not found `. I'm adding a user attribute to check if error is being fixed or not

Reply:
@iamareebjamal the tests passed. I used both session and user (which are currently into account). When we will kove to event rating, the tests will be updated and EventFactorybasic will be included. Please have a look now

Reply:
Instead of breaking a functionality, add it in such a way it does not break previous one. Add a new endpoint in API Blueprints instead of changing existing one

Reply:
@iamareebjamal event feedback functionality is non-existent. 
There was no check or piece of code which was for event rating functionality in our system. There was only feedback schema with both event and session relationships, so I just used it for session rating. When event rating is to be implemented, we can create a separate API blueprint for it at that time. 
However if you think that I should revert the changes and create a separate API blueprint, please let me know. I'll make changes accordingly if you want

Reply:
The same API was being used for event feedback if I'm not wrong

Reply:
@iamareebjamal actually there is no event feedback in existence AFAIK. However I've asked others on gitter if it exists.
There was only feedback schema and models which were meant for future implementation. They were never used (atleast in my knowledge).
Is android using it already?